sdk/playwrighttesting/microsoft-playwright-testing/src/utils/utils.ts

```diff
@@ -98,6 +99,20 @@ export const validateMptPAT = (
     if (result!.accountId !== claims!.aid) {
       validationFailureCallback(ServiceErrorMessageConstants.WORKSPACE_MISMATCH_ERROR);
     }
+    const currentTime = Date.now();
+    const expirationTime = claims.exp! * 1000;
+    const sevenDaysInMs = 7 * 24 * 60 * 60 * 1000;
+
+    if (
+      expirationTime - currentTime <= sevenDaysInMs &&
+      process.env[oneTimeEnviromentVariable.WARNING_MPT_PAT_CLOSE_TO_EXPIRY] !== "true"
```

Comment:
Let's create a single environment variable that will run once for every setup instead of creating multiple environment variables for the same use-case. We can then move the token validation check in the getServiceConfig() method.